changelog.md

```diff
@@ -0,0 +1,116 @@
+# Microsoft Orleans Changelog
+All notable end-user facing changes are documented in this file.
+
+### [vNext]
+*Here are all the changes in `master` branch, and will be moved to the appropriate release once they are included in a published nuget package.
+The idea is to track end-user facing changes as they occur.*
+
+- Sample change. We should start adding items here when we submit PRs
+
+
+
+### [v1.2.2]
+- Bugfix: Remote stacktrace is once again being included in the exception that bubbles up to the caller (bug introduced in 1.2.0). #1808
```

Comment:
haha, that I don't know yet... I considered having the full link, but if it's not required, I would strongly support the short version. For example in the releases tab in github, we are using full links, but the short ones work fine (but there it's not a big deal, as we don't expect users to see the raw markdown)